draft-ietf-quic-http.md

```diff
@@ -1155,22 +1151,25 @@ The PRIORITY frame payload has the following fields:
 
   PT (Prioritized Element Type):
   : A two-bit field indicating the type of element being prioritized (see
-    {{prioritized-element-types}}). When sent on a request stream, this MUST be
-    set to `11`.  When sent on the control stream, this MUST NOT be set to `11`.
```

Comment:
We probably want to retain the "MUST NOT be set to `11` here, even if we call it out as an error later.

Reply:
Makes sense. Done in 0fd27179.